Lowest Common Ancestor of Binary Tree

```diff
@@ -0,0 +1,45 @@
+//TC: O(n)
+//SC: O(m+n) ??
+
+//Approach: Idea is to use backtracking to create path having only elements from root to target and remove other traversed elements
```

Comment:
Please try to optimize on this algorithm. Try to see if normal recursion can be used.